components/LostItem.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import Image from "next/image";
 import styled from "styled-components";
+import Link from "next/link";
 
 export default function LostItem({ name, description, img, id }) {
   return (
-    <div>
+    <Link href={`/details/${id}`}>
```

Comment:
Same feedback as above. Please check your link should really wrap the whole content.